server/src/main/java/org/opensearch/cluster/decommission/DecommissionController.java

```diff
@@ -267,4 +271,41 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
             }
         });
     }
+
+    void setWeights(String awarenessAttribute, Map<String, Double> weights, ActionListener<ClusterPutWeightedRoutingResponse> listener) {
```

Comment:
This will be generic method. For Recommission and graceful node decommission. Tests is already added as part of GracefulNodeDecommission PR

https://github.com/opensearch-project/OpenSearch/pull/4586

Reply:
Yes. we will be removing this change.. Actually merging with latest should resolve this.